### PR TITLE
[DX] Potential missing annotation route import

### PR DESCRIPTION
**Pull request by benji07**
Register controller route when a bundle is generated in xml, yml and php, but we want to generate a crud with annotation

### CODE REVIEW

File: Manipulator/RoutingManipulator.php [hunk 1]
```diff
@@ -74,4 +75,63 @@ public function addResource($bundle, $format, $prefix = '/', $path = 'routing')
...
+        }
+
+        $code = sprintf("%s:\n", Container::underscore(substr($bundle, 0, -6)).'_'.Container::underscore($controller));
+
+        $code .= sprintf("    resource: \"@%s/Controller/%sController.php\"\n    type:     annotation\n", $bundle, $controller);
+
+        $code .= "\n";
+        $code .= $current;
+
+        if (false === file_put_contents($this->file, $code)) {
```
Comment (phansys) [inline Manipulator/RoutingManipulator.php]: Why not just ```return false !==  file_put_contents($this->file, $code);```?

File: Manipulator/RoutingManipulator.php [hunk 2]
```diff
@@ -74,4 +75,63 @@ public function addResource($bundle, $format, $prefix = '/', $path = 'routing')
...
+            return false;
+        }
+
+        $config = Yaml::parse($this->file);
+
+        $search = sprintf('@%s/Controller/', $bundle);
+
+        foreach ($config as $resource) {
+            if (array_key_exists('resource', $resource)) {
+                return $resource['resource'] == $search;
```
Comment (phansys) [inline Manipulator/RoutingManipulator.php]: Strict comparison should be used.

File: Manipulator/RoutingManipulator.php [hunk 3]
```diff
@@ -74,4 +75,63 @@ public function addResource($bundle, $format, $prefix = '/', $path = 'routing')
...
+     *
+     * @return bool
+     */
+    public function hasResourceInAnnotation($bundle)
+    {
+        if (!file_exists($this->file)) {
+            return false;
+        }
+
+        $config = Yaml::parse($this->file);
```
Comment (phansys) [inline Manipulator/RoutingManipulator.php]: >Passing a filename is deprecated in Symfony 2.2, and will be removed in Symfony 3.0.
[[Ref]](http://symfony.com/doc/current/components/yaml/introduction.html).

**Comment by javiereguiluz** [comment]
@benji07 thanks for your great work on this. Since only a final push was needed to finish it, I've decided to create a new PR in #403. I reused all your commits, so you'll get full credit on your work. Thanks!

**Comment by benji07** [comment]
@javiereguiluz :+1:

**Pull request closed without merging**